DotNetRu.Clients.UI/DotNetRu.Clients.UI.csproj

```diff
@@ -77,7 +77,6 @@
     <Compile Include="Pages\iOS\AboutPage.xaml.cs">
       <DependentUpon>AboutPage.xaml</DependentUpon>
     </Compile>
-    <Compile Include="Helpers\Settings.cs" />
```

Comment:
Why we remove it? We need it to save lang settings

Reply:
There was several settings file (one in each project), I've removed unneeded ones.
Language settings are saved in different Settings.cs file